tests/data/parsers/test_parser_classification_rule.py

```diff
@@ -202,7 +202,7 @@ def test_feedback_for_criteria_with_gaps_and_overlap(
                 [7, ">=0", ">=5", "<10"],
                 [8, ">=0", ">=5", ">=10"],
             ],
-            "",
+            [call("")],
```

Comment:
why this change? What does mock.call do exactly?

Reply:
this changed because I needed to check on multiple calls. One for the warnings and one for the information about too many warnings.. but if that last one isn't called you only get one back.. but because of the parametrized functionality this test is updated as well